aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -10,7 +10,7 @@
     /* global Cl, describe, it, expect */
 
     // ########################################################################
-    describe('cl.events.js', function () {
+    describe('cl.events.js:', function () {
```

Comment:
do we need the colon? The sentence should read "cl.events.js has available Cl namespace"?